YPP-0103.md

```diff
@@ -0,0 +1,197 @@
+# Proposal
+
+Increase the dust amd max levels for FCash.
+
+# Background
+
+The maximum borrow for fCash's USDC/FUSDC and DAI/FDAI borrowing pairs was previously set at 500k USDC. This limit was quickly reached in the USDC/FUSDC pair. We plan to raise this limit to 1000000 (1 million) USDC. Similarily, we will raise the DAI limit to the same amount.
+
+Also, the dust level for fCAsh FUSDC was set to 5000 (USD value). We will lower it to 1000. This was level was previoulsy lowered for all other borrowing pairs (YPP-0086). 
+
+(The limits for ETH/FETH were correct and will not be changed)
+
+# Details
+
+The [updateLimits.ts]
+
+(https://github.com/yieldprotocol/environments-v2/pull/293/files#diff-eff1c2a858cba3323fcc396b80daa2aad677ce28500f9825ddcf7a4e1bc29856) 
+
+script will be used, with this input:
+
+```
+/// @notice Limits to be used in an auction
+/// @param base identifier (bytes6 tag)
+/// @param ilk identifier (bytes6 tag)
+/// @param Maximum global debt allowed for this pair, modified by decimals
+/// @param Minimum vault debt allowed for this pair, modified by decimals
+/// @param Decimals to append to global and vault debt.
+export const newLimits: Array<[string, string, number, number, number]> = [
+  [DAI, FDAI2303, 1000000, 1000, 18],
+  [DAI, FDAI2306, 1000000, 1000, 18],
+  [USDC, FDAI2303, 1000000, 1000, 6],
+  [USDC, FDAI2306, 1000000, 1000, 6],
```

Comment:
This doesn't reflect the [actual config](https://github.com/yieldprotocol/environments-v2/blob/b3a1edf99d8f30feaae74e48168d37d3e6444c99/scripts/governance/update/updateLimits/updateLimits.mainnet.config.ts#L39), where USDC relates to FUSDC ilks.

Reply:
fixed